clusterloader2/cmd/clusterloader.go

```diff
@@ -127,7 +127,14 @@ func main() {
 	for _, clusterLoaderConfig.TestConfigPath = range testConfigPaths {
 		glog.Infof("Running %v", clusterLoaderConfig.TestConfigPath)
 		if errList := test.RunTest(f, &clusterLoaderConfig); !errList.IsEmpty() {
-			glog.Fatalf("Test execution failed: %v", errList.String())
+			if metricErrList := errList.Filter(errors.IsMetricViolationError); !metricErrList.IsEmpty() {
+				glog.Errorf("Test slos failed: %v", errList.String())
+			}
+			// TODO: Reimplement execution errors handling.
+			if executionErrList := errList.Filter(func(e error) bool { return !errors.IsMetricViolationError(e) }); !executionErrList.IsEmpty() {
```

Comment:
Can we introduce IsNotMetricViolationError helper?

Reply:
Done.